Library/Formula/ddar.rb

```diff
@@ -9,7 +9,7 @@ class Ddar < Formula
 
   depends_on "xmltoman" => :build
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on "protobuf"
```

Comment:
```
$ brew audit ddar
ddar:
 * Dependency protobuf does not define option "with-python"
```

Since #39260 in 5/2015, `protobuf` now has a `--without-python` option; including it is the default.